server/src/modules/user/user.controller.ts

```diff
@@ -354,9 +354,8 @@ export class UserController {
     })
     @ApiNoContentResponse({ description: 'Map was added to the library' })
     @ApiNotFoundResponse({ description: 'The map does not exist' })
-    addMapLibraryEntry(@LoggedInUser('id') _userID: number, @Param('mapID', ParseIntPipe) _mapID: number) {
-        return void 0;
-        //return this.usersService.AddMapLibraryEntry(userID, mapID);
+    addMapLibraryEntry(@LoggedInUser('id') _userID: number, @Param('mapID', ParseIntPipe) _mapID: number){
+        return this.usersService.addMapLibraryEntry(_userID, _mapID);
```

Comment:
I was underscoring the variable names here as they were unused (so linter would shout at us), but now they're being used, no need for the underscores